### PR TITLE
Add option to hide author email

### PR DESCRIPTION
**Pull request by benlk**
## Changes

- Adds checkbox to the user edit page to change whether the author email address is displayed publicly. This defaults to showing the author's email address.
- Uses that user meta to toggle whether the email button is saved in the author bio social links partial.

screen shot 2015-11-05 at 1 27 34 pm

## Why

For #659 and [WE-1](jira.inn.org/browse/WE-1)

This is useful for authors that want to keep their email address private and for accounts that are managed by many people.

### CODE REVIEW

**Comment by aschweigert** [comment]
needed for Women's eNews

**Comment by benlk** [comment]
This is bugged: The email address shown is '1' on the author archive page.

Possibly related: No widgets are shown in the article-bottom widget area!?

File: partials/author-bio-social-links.php [hunk 1]
```diff
@@ -11,7 +25,7 @@
 		</li>
 	<?php } ?>
 
-	<?php if ( $email = $author_obj->user_email ) { ?>
+	<?php if ( $email = $author_obj->user_email && $show_email !== 'off' ) { ?>
```
Comment (benlk) [inline partials/author-bio-social-links.php]: This line somehow sets $email to 'true'.

**Comment by benlk** [comment]
The email address displayed is no longer '1'.

**Pull request merged**